Alignment/APEEstimation/python/MisalignmentScenarios_cff.py

```diff
@@ -1238,3 +1238,30 @@
         ),
     ),
 )
+
+MisalignmentScenario_PhaseI_PseudoAsymptotic = cms.PSet(
```

Comment:
@ghellwig please add a reference (twiki page) from where this numbers are taken from.